test/distributed/_tensor/test_dtensor_compile.py

```diff
@@ -270,6 +281,49 @@ def test_2d_fsdp_tp_compile(self):
 
         self.assertEqual(out, compiled_output)
 
+    @with_comms
+    @skip_if_lt_x_gpu(4)
+    def test_2d_fsdp_tp_ac_compile(self):
+        dp_degree = 2
+        tp_degree = self.world_size // dp_degree
+        model = SimpleModel(self.device_type)
+        model_copy = copy.deepcopy(model)
+        enable_2d_with_fsdp()
```

Comment:
We no longer need `enable_2d_with_fsdp` anymore with the device_mesh approach.

Reply:
sg, I'll delete it